### PR TITLE
Implement the creation of multiple members with a batch.

### PR DESCRIPTION
**Pull request by ivanst-stoyanov**
As promised I have implemented the creation of multiple members with a batch.
I have also implemented the creation of lists.

### CODE REVIEW

File: src/main/java/com/mailchimp/domain/ListDefault.java [hunk 1]
```diff
@@ -22,9 +25,93 @@
...
+    }
+
+    @JsonProperty(value = JsonConstants.CAMPAIGN_DETAILS)
+    private CampaignDefaults campainDetails;
+
+    public CampaignDefaults getCampainDetails()
+    {
+        return campainDetails;
+    }
+
```
Comment (micheal-swiggs) [inline src/main/java/com/mailchimp/domain/ListDefault.java]: Spelling please.

File: src/main/java/com/mailchimp/domain/MemberCreate.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package com.mailchimp.domain;
```
Comment (micheal-swiggs) [inline src/main/java/com/mailchimp/domain/MemberCreate.java]: Hmmm is it possible to combine the changes of this class into MemberDefault?

File: src/main/java/com/mailchimp/domain/MemberDefault.java [hunk 3]
```diff
@@ -12,29 +12,38 @@
...
     @JsonProperty(value = JsonConstants.ID)
     private String id;
 
+    public String getId()
+    {
+        return id;
+    }
+
     @JsonProperty(value = JsonConstants.MEMBER_RATING)
     private Integer memberRating;
```
Comment (micheal-swiggs) [inline src/main/java/com/mailchimp/domain/MemberDefault.java:28]: Is it not possible to use the getter provided by lombok?

**Comment by micheal-swiggs** [comment]
@ivanst-stoyanov Thanks for this I'm just going through this. Looks great only small styling things that I can see at the moment. Is it possible to use the `getters` provided by lombok? Thanks.

**Comment by ivanst-stoyanov** [comment]
@micheal-swiggs  Fixed it. This is the first time I use Lombok, so I had to configure Intellij to support it. Now everything works great.

**Comment by micheal-swiggs** [comment]
Thanks @ivanst-stoyanov merged d60c2c096

**Pull request closed without merging**